mythril/ether/util.py

```diff
@@ -18,10 +18,17 @@ def safe_decode(hex_encoded_string):
 
 def get_solc_json(file, solc_binary="solc", solc_args=None):
 
-    cmd = [solc_binary, "--combined-json", "bin,bin-runtime,srcmap,srcmap-runtime", '--allow-paths', "."]
+    cmd = [solc_binary, "--combined-json", "bin,bin-runtime,srcmap,srcmap-runtime"]
 
     if solc_args:
-        cmd.extend(solc_args.split(" "))
+        cmd.extend([i for i in solc_args.split(" ") if i != ""])
```

Comment:
I think you can remove the " " in the solc_args.split(" "), and make it `solc_args.split()` and then you don't need the check for `i != ""` .

Reply:
Good point, I haven't written Python code in a while :) - updated in 6fd8b76900bc2098e6007b126e5eb2995f43b6d1 (rebased)